deployment/charts/templates/deploy.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.app.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-service-monitor
+  labels:
+    release: {{ .Values.app.prometheus.helm_release_name }}
+spec:
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
+  endpoints:
+    - path: /metrics
+      port: http
+{{- end }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fuel-block-committer.name" . }}
+    chart: {{ template "fuel-block-committer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "fuel-block-committer.name" . }}-service
+spec:
+  type: NodePort
```

Comment:
Not everyone wants to use NodePort. If you have nodes only in private networks, those will need to be exposed through a load balancer. Better give the option to the user to specify what the type of service

Reply:
will leave this to a followup

Reply:
https://github.com/FuelLabs/fuel-block-committer/issues/42